src/ralph/defaults.py

```diff
@@ -107,6 +107,8 @@ def load_config(config_file_path):
 
 APP_DIR = Path(environ.get("RALPH_APP_DIR", get_app_dir("ralph")))
 DEFAULT_ENCODING = getattr(io, "LOCALE_ENCODING", "utf8")
+DEFAULT_RUNSERVER_HOST = "0.0.0.0"
+DEFAULT_RUNSERVER_PORT = 8100
```

Comment:
Why not removing the default value in `click` and add a config file to those variable? Hence, they can also be set in the `env` file.

Reply:
Removing the default value? You mean removing the options from click and just using the environment for this?